### ISSUE TITLE
[Feature]: Developer gets Flask app failure feedback from cf logs and logs of Korifi API

### ISSUE THREAD

**Issue by adamfowleruk** [cloudfoundry/korifi#1733]
### Background

**As a** Developer pushing a basic Flask app
**I want** feedback when I've forgotten basic config (E.g. Procfile) causing the container not to start repeatedly
**So that** I can diagnose my personal memory failure and resolve my app woes more quickly

Not entirely sure why this is not in the logs, but given cf push hangs starting the app for 120 seconds and there are hundreds of restarts in the logs, we need to ensure we're doing a better job reflecting these failures in the cf push command and in cf logs without dropping 'under the hood' of cf and describing Kubernetes objects (which actually didn't help me in any case in this instance - but at least I saw what stage the failure was at).

### Acceptance Criteria

**GIVEN** a cf push of a Python Flask API app that fails on start
**WHEN I** execute cf logs for the app, or view the Korifi logs, or describe the underlying Kubernetes resources
**THEN I** see the container failure message from container start up reported

### Dev Notes

I can try and reproduce the same failure locally to see what log output should be possible, then report back with what the Korifi team need to capture once I've discovered it, if that helps.

**Comment by danail-branekov** [comment]
Hey @adamfowleruk 

Could you provide a sample app that reproduces the issue (we are not familiar with Flask)

> there are hundreds of restarts in the logs
which logs are you referring to?

**Comment by georgethebeatle** [comment]
We went through the following steps:
- Created a flask app as follows (app.rb):

```python
from flask import Flask

app = Flask(__name__)

@app.route("/")
def hello_world():
    return "<p>Hello, World!</p>"
```
- Added the `gcr.io/paketo-buildpacks/python` to both the `ClusterStore` and `ClusterBuilder`
- Pushed the flask app

Indeed the app did not start and `cf logs` did not display anything useful, while the pod was in a `CrashLoopBackoff` state. - The pod's logs were empty 
- Thee app container had an exit code of 0
- The app command was `/cnb/lifecycle/launcher python`

We read a bit more about the python buildpack and found out that if a framework like `flask` or `gunicorn` is being used, there needs to be a `Procfile` that contains the proper startup command for the framework.

We included the following Procfile with our `flask` app:

```
web: flask --app app run
```

When we pushed the updated app it failed again, but this time `cf logs` contained a log line as follows:
```
[APP/] OUT bash: flask: command not found
```
This is normal, since the clusterstack we are using does not include the flask runtime.

In conclusion the problem you are observing is very specific to the python buildpack, since it defaults to `python` as a command unless you give it a `Procfile` and `python` when run it non-interactive mode exits 0 and does not print anything to stderr. This is why the pod has an empty log and `cf logs` does not give any useful information. 

Closing,  as this does not seem to be a bug in Korifi. Feel free to open if you have any ideas on how we can improve this.
